opentelemetry-sdk/src/opentelemetry/sdk/metrics/__init__.py

```diff
@@ -186,72 +235,86 @@ def __init__(
         description: str,
         unit: str,
         value_type: Type[metrics_api.ValueT],
+        meter: "Meter",
         label_keys: Sequence[str] = (),
         enabled: bool = True,
         monotonic: bool = False,
+        absolute: bool = False,
     ):
         super().__init__(
             name,
             description,
             unit,
             value_type,
+            meter,
             label_keys=label_keys,
             enabled=enabled,
             monotonic=monotonic,
+            absolute=absolute,
         )
 
-    def set(self, label_set: LabelSet, value: metrics_api.ValueT) -> None:
+    def set(self, value: metrics_api.ValueT, label_set: LabelSet) -> None:
         """See `opentelemetry.metrics.Gauge.set`."""
         self.get_handle(label_set).set(value)
 
     UPDATE_FUNCTION = set
 
 
 class Measure(Metric, metrics_api.Measure):
-    """See `opentelemetry.metrics.Measure`.
-
-    By default, measure metrics can accept both positive and negatives.
-    Negative inputs will be discarded when monotonic is True.
-    """
+    """See `opentelemetry.metrics.Measure`."""
 
     HANDLE_TYPE = MeasureHandle
 
-    def __init__(
```

Comment:
Why removing the constructor here?

Reply:
Measure can use `Metric`, the parent class as a constructor since we separated the `monotonic` and `absolute` parameters.